README.md

```diff
@@ -1,30 +1,55 @@
+**Persistent cookies** allow a website to remember your user information and preferences for *future* visits. A persistent cookie is stored on your computer, while a session cookie is temporarily stored in your web browser. Persistent cookies allow for faster page loading, automatic login and user authentication, and access to other potential web application features because much of the data that these features require will be hanging out in a persistent cookie, stored on your computer. So, you don't have to go through the cycle of sending a web request to a side and waiting to receive data back from the server in order to for your browser to have your log in info, for example. Persistent cookies are usually created on the first visit to the page after you have created an account, and typically don't expire unless the web application has a protocol in place for cookie expiration, or the user clears their browser's cookie cache.
 
-Session cookies allow the website to keep track of your movement from page to page for that specific session (from the time you log in to the time you log out/close the browser). Session cookies simply allow you to navigate through a site without repeatedly having to authenticate yourself on every new page you visit within the web application domain. Session cookies expire every time out log out or navigate away from the website.
+What would happen if we didn't have persistent cookies? We would have to go to Facebook's login page and type in our email address and passwords every single time we wanted to visit Facebook. Our user settings and preferences would not be remembered, and the quality of the user's experience would go way down.
 
-Let's walk through an example of what would happen without a session cookie. If you were on Amazon, added an item to your shopping cart, and then selected the `Checkout` option, the new page would have no recollection of your prior activity. Your shopping cart would always be empty, even though you explicitly added an item earlier. There would be no record of your activities for that session.
+## Coming Up...
 
-Persistent cookies allow the website to remember your user information and preferences for future visits. This allows for faster page loading, automatic login and user authentication, and access to other potential web application features. Persistent cookies are usually created on the first visit to the page after you have created an account, and typically don't expire unless the web application has a protocol in place for cookie expiration, or the user clears their browser's cookie cache.
+So far, we've defined a **session** as a hash that lives on the server side of our application and stores information about a particular user who is interacting with our website at a given point in time. The session object will get information from and interact with cookies. Later on, we'll learn how to use the session object to build apps that allow users to log in and log out. 
```

Comment:
Does this make sense?

Cookies are client side counterparts to sessions. They store information on client computers and and are visible only to the server that created them. Client browsers send cookies to remote servers along with each request, then the server can read the cookie, associate it with an existing session if any and decides how to respond accordingly. (e.g. if there is no cookie received: show login page, cookie received but doesn't match a session: show login page remembering user name, cookie received and matches a session : respond with data). 
Later on, we'll learn how to use the cookie and session objects to build apps that allow users to log in and log out.